README.md

```diff
@@ -444,7 +444,6 @@ A method that calls Firebase's [`signOut`](https://firebase.google.com/docs/refe
 
 We expect some apps will need some additional customization that's not currently available:
 
-* **Dynamic redirect destinations:** Currently, built-in *dynamic* redirects aren't possible, because `authPageURL` and `appPageURL` are static. We may want to allow a function so the app can determine the redirect destination based on, for example, a URL parameter value.
```

Comment:
We should keep this, because this problem still isn't resolved. *Built-in* redirects still aren't supported, and there's not a clean way to do client-side redirects. Maybe change to:

**Built-in, dynamic redirect destinations:** Currently, built-in, dynamic redirects aren't possible, because `authPageURL` and `appPageURL` are static. However, dynamic server-side redirects are possible by return a "redirect" key from the function passed to `withAuthUserTokenSSR`. We may want to allow a function so the app can determine the redirect destination based on, for example, a URL parameter value.

Reply:
hi @kmjennison ... I'll take a look at the enhancement proposal and give you feedback sometime today

Reply:
> t

I agree, the custom redirect logic only applies to SSR, I will update accordingly.